### PR TITLE
[Unique-10] Add Unique op to PythonAPI

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
 - Add Unique-10 op to PythonAPI
 - Add Unique-10 op to compatibility PythonAPI
 
### Tickets:
 - 94714

### CODE REVIEW

File: src/bindings/python/src/openvino/runtime/opset10/ops.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from openvino.runtime.opset_utils import _get_node_factory
 from openvino.runtime.utils.decorators import nameable_op
 from openvino.runtime.utils.types import (
+    NumericType,
```
Comment (jiwaszki) [inline src/bindings/python/src/openvino/runtime/opset10/ops.py:13]: Should it be here? I do not see a use of it in changes.

**Review by jiwaszki — APPROVED** [review]
Overall looks good!

**Review by mmikolajcz — APPROVED** [review]
Looks good

**Review by p-wysocki — APPROVED** [review]

File: src/bindings/python/src/openvino/runtime/opset10/ops.py (hunk 1 above)
Comment (mitruska) [reply src/bindings/python/src/openvino/runtime/opset10/ops.py]: Removed

**Pull request merged**